pyproject.toml

```diff
@@ -0,0 +1,102 @@
+# WARNING: This file is used only for development done on this addon.
+# Be aware that dependencies used here might not match the ones used by
+# the specific addon bundle set up on the AYON server. This file should
+# be used only for local development and CI/CD purposes.
+
+[tool.poetry]
+name = "ayon-core"
+version = "0.3.0"
```

Comment:
I think new poetry will bitch about it because it doesn't follow https://packaging.python.org/en/latest/specifications/version-specifiers/#version-specifiers